draft-ietf-core-coap-tcp-tls.md

```diff
@@ -716,113 +668,7 @@ Connection, then the CoAP server (or intermediary in the role of
 the CoAP server) MUST remove all entries associated with the client
 from the lists of observers when the connection is closed.
 
-# Reliable Transports and Block-Wise Transfer

```

Comment:
I did not check whether this just was moved or whether there were changes as well -- generally it is impossible to work on PRs that mix moves and changes.